data/src/main/java/com/teamagam/gimelgimel/data/message/poller/MessageLongPoller.java

```diff
@@ -44,18 +40,20 @@ public MessageLongPoller(GGMessagingAPI messagingAPI, MessageDataMapper messageD
   public Observable poll() {
     long synchronizedDateMs = mPrefs.getLong(Constants.LATEST_MESSAGE_DATE_KEY);
 
-    return poll(synchronizedDateMs).map(this::getMaximumDate)
+    return poll(synchronizedDateMs)
+        .toFlowable(BackpressureStrategy.BUFFER)
```

Comment:
Nice, but I think we should solve this problem from its root.
The `GGMessaingAPI` should return a `Flowable` and not `Observable`.